google/cloud/spanner/internal/session_pool.cc

```diff
@@ -144,10 +144,14 @@ std::shared_ptr<SpannerStub> SessionPool::GetStub(Session const& session) {
   return stub;
 }
 
-void SessionPool::Release(Session* session) {
+void SessionPool::Release(std::unique_ptr<Session> session) {
   std::unique_lock<std::mutex> lk(mu_);
+  if (session->is_bad()) {
+    --total_sessions_;
```

Comment:
Opt: I believe we wanted a TODO here to remind us to signal someone to replenish the lost session.

Reply:
Done